tests/testAutopas/ParticleIteratorTest.cpp

```diff
@@ -24,6 +24,12 @@ void ParticleIteratorTest::SetUp() {
 
 void ParticleIteratorTest::TearDown() {}
 
+#ifdef AUTOPAS_OPENMP
+// reduction for merging vectors: {1,2} + {2,3} -> {1,2,2,3}
+#pragma omp declare reduction(vecMerge : std::vector<size_t> : omp_out.insert(omp_out.end(), omp_in.begin(), omp_in.end()))
```

Comment:
very nice reduction operation. we should remember this for other parts of the code.

Reply:
probably doesn't work for simd, right?

Reply:
We could move this to WrapOpenMP.h